AEPCampaignClassic/Sources/RegistrationManager.swift

```diff
@@ -13,6 +13,7 @@
 import Foundation
 import AEPServices
 import AEPCore
+import UIKit
```

Comment:
We use UIDevice class to grab OS information. Thus we need to import UIKit.
(This was  shown as an error while using AEPCampaignClassic using swift package)